jobs/v3/howto/basic_company_sample.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sample
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	talent "google.golang.org/api/jobs/v3"
+)
+
+// [START create_service]
+
+// createCtsService creates service of Cloud Talent Solution.
+func createCtsService() (*talent.Service, error) {
+	// Authorize the client using Application Default Credentials.
+	// See https://g.co/dv/identity/protocols/application-default-credentials
+	ctx := context.Background()
+	client, err := google.DefaultClient(ctx, talent.CloudPlatformScope)
+	if err != nil {
+		log.Fatalf("Failed to create client of Cloud Talent Solution, Err: %v", err)
+	}
+	// Create the jobs service client.
+	ctsService, err := talent.New(client)
+	if err != nil {
+		log.Fatalf("Failed to create service of Cloud Talent Solution, Err: %v", err)
+	}
+	return ctsService, err
+}
+
+// [END create_service]
+
+// [START basic_company]
+
+// constructCompanyWithRequiredFields constructs a company with required fields: ExternalId and DisplayName.
+func constructCompanyWithRequiredFields() *talent.Company {
```

Comment:
Is this function needed? In the documentation, this function won't be shown next to the snippet, so it will make it more difficult to understand.